docs/content/envoy-primer.md

```diff
@@ -208,6 +208,7 @@ When Envoy receives a policy decision, it expects a JSON object with the followi
 * `body` (optional): the response body
 * `dynamic_metadata` (optional): an object representing dynamic metadata to be consumed by the next Envoy filter.
 * `query_parameters_to_remove` (optional): is an array containing the names of string query parameters to be removed.
+* `query_parameters_to_set` (optional): an object mapping parameter names to values (string) or arrays of values (for multiple values with the same key)
```

Comment:
Probably we can add a longer description in https://www.openpolicyagent.org/docs/latest/envoy-primer/#example-policy-with-additional-controls and then a brief one here. Same for `query_parameters_to_remove` if you can do that as well. Thanks